odoo/addons/base/models/res_country.py

```diff
@@ -37,7 +42,10 @@ class Country(models.Model):
              "(in reports for example), while this field is used to modify the input form for "
              "addresses.")
     currency_id = fields.Many2one('res.currency', string='Currency')
-    image = fields.Binary(attachment=True)
+    image_url = fields.Char(
+        compute="_compute_image_url_from_country_code", string="Flag",
```

Comment:
Guidelines are usually "_compute_image_url".
The fact that it's from the country code is an implementation detail and easily deduced by the "depends".